toolbox/fdc3-conformance/Intents-Tests.md

```diff
@@ -0,0 +1,240 @@
+- `2.0-FindIntentAppDByResultChannel1`: Calls `fdc3.findIntent("sharedTestingIntent2", testContextY, "channel")`.  Receives promise containing an `AppIntent` with metadata containing `sharedTestingIntent2` and only **E** and **F** app metadata.
+- `2.0-FindIntentAppDByResultChannel2`: Calls `fdc3.findIntent("sharedTestingIntent2", testContextY, "channel<testContextZ>")`.  Receives promise containing an `AppIntent` with metadata containing `sharedTestingIntent1` and only **F** app metadata.
+
+### Raise Intent (Ignoring any result)
+
+| App   | Step           | Details                                                                                           |
+|-------|----------------|---------------------------------------------------------------------------------------------------|
+| Test  | 1. Raise        | `fdc3.raiseIntent(‘aTestingIntent1’, testContextX)`<br>starts app A.                       |
+| A     | 2. Receive Intent & Context | After starting up, A runs `fdc3.addIntentListener(‘aTestingIntent1’)` to register its listener.<br>It then receives `testContextY`, matching that sent by Test |
+| Test  | 3. IntentResolution          | The `raiseIntent` call returns an `IntentResolution` Object with an `AppIdentifer` as the `source field` with App A's `appId` and `instanceId` set.**                       |
+
+- `2.0-RaiseIntentSingleResolve`: Perform above test
+- `2.0-RaiseIntentTargetedAppResolve`: Repeat the above test, but: 
+  - In the first step use `fdc3.raiseIntent(‘aTestingIntent’, testContextY, {"appID": "<Y’s appId>"})` to start app A, 
+  - Otherwise, as above.
+- `2.0-RaiseIntentTargetedInstanceResolveOpen`: Repeat the above test, but:** 
+  - Before the first step, use `let appIdentifier = await fdc3.open({appId: "<A's appId>"})` to start A and retrieve its `AppIdentifier` with instance details.
+  - Then in the first step, use `fdc3.raiseIntent(‘aTestingIntent’, testContextX, appIdentifier)` to target the running instance of app A, otherwise, as above.
+- `2.0-RaiseIntentTargetedInstanceResolveFindInstances`: Repeat the above test, but:
+  - Before the first step, use `let appIdentifier = await fdc3.open({appId: "<A's appId>"})` to start A and retrieve its `AppIdentifier` with instance details.
+  - Then in the first step, use `fdc3.raiseIntent(‘aTestingIntent’, testContextX, appIdentifier)` to target the running instance of app A, otherwise, as above.
+- `2.0-RaiseIntentFailedResolve`: Perform above test, but:
+  - Use `fdc3.raiseIntent(‘aTestingIntent’, testContextY)`.  Note that no app supports this intent and context combination.**
+  - You should receive a JavaScript Error with the message `ResolveError.NoAppsFound`.
+- `2.0-RaiseIntentFailTargetedAppResolve1`: Perform above test, but:** 
```

Comment:
in `2.0-RaiseIntentFailTargetedAppResolve1`: `ResolveError.NoAppsFound` needs to be changed to `ResolveError.IntentDeliveryFailed`.